website/static/js/contribAdder.js

```diff
@@ -80,6 +79,26 @@ var AddContributorViewModel = oop.extend(Paginator, {
             return self.query() && !self.results().length;
         });
 
+        self.addAllVisible = ko.pureComputed(function() {
+            var selected_ids = self.selection().map(function(user){
+                return user.id;
+            });
+            for(var i = 0; i < self.results().length; i++) {
+                if(self.contributors().indexOf(self.results()[i].id) === -1 &&
```

Comment:
We only want to return true if one of ids is not in the two lists. Then outside of the for loop it returns false. The suggested simplification would work if it didn't need to go through the list.